examples/models/transform_jitter.py

```diff
@@ -15,7 +15,7 @@
 normal = Jitter(width=0.2, distribution="normal")
 uniform = Jitter(width=0.2, distribution="uniform")
 
-p = figure(x_range=(0, 4), y_range=(0,10))
+p = figure(x_range=(0, 4), y_range=(0,10),toolbar_location='right')
```

Comment:
@andreagrant I think @birdsarah was referring to the fact that a PR was merged a few days ago that makes the right side the new default location.